CHANGELOG.md

```diff
@@ -4,43 +4,14 @@ Anticipated release: October 26th, 2020
 
 #### 🚀 New features
 
-- Add a label to the rich text editor `<textarea>` element ([#2546])
-- Metrics: Research using Google Analytics for Operational Metrics ([#2139])
-- Metrics: Create APD event database table ([#2140])
-- Metrics: API endpoint for Export events ([#2141])
-- Metrics: Hook the export button into the API ([#2142])
-- Replace the “Cost Allocation and Budget for FFY 2020” table with part of the new "Summary Budget by Activity” table ([#2306])
-- Other funding amount should be represented as a subtraction on Activity Breakdown table ([#2429])
-- Where groups of checkboxes are used, use appropriate grouping markup including legend/label/fieldset ([#2545])
-- Updates tables to increase accessibility ([#2501])
-- Updates markup to use semantic `<nav>` and `<main>` where appropriate ([#2502])
-- Updated inputs in the Estimated Quarterly Incentive Payments table to use aria-labelledby for better UX/a11y ([#2503])
-- Give users the ability to choose FFYs before entering their Key Personnel ([#2509])
-- Add bulleted lists, numbered lists, formatting dropdown to the RichText editor ([#2521])
+- Update areas in the app where "total computable" should be used ([#2324])
 
 #### 🐛 Bugs fixed
 
 #### ⚙️ Behind the scenes
 
-- Update @cmsgov/design-system to v2.1.1 [[releases](https://github.com/CMSgov/design-system/releases)]
-- Refactor use of Dropdown component ([#2471])
-
 # Previous releases
 
 See our [release history](https://github.com/CMSgov/eAPD/releases)
 
-[#2471]: https://github.com/CMSgov/eAPD/issues/2471
-[#2322]: https://github.com/CMSgov/eAPD/issues/2322
-[#2546]: https://github.com/CMSgov/eAPD/issues/2546
-[#2139]: https://github.com/CMSgov/eAPD/issues/2139
-[#2140]: https://github.com/CMSgov/eAPD/issues/2140
-[#2141]: https://github.com/CMSgov/eAPD/issues/2141
-[#2142]: https://github.com/CMSgov/eAPD/issues/2142
-[#2306]: https://github.com/CMSgov/eAPD/issues/2306
-[#2429]: https://github.com/CMSgov/eAPD/issues/2429
-[#2545]: https://github.com/CMSgov/eAPD/issues/2545
-[#2501]: https://github.com/CMSgov/eAPD/issues/2501
-[#2502]: https://github.com/CMSgov/eAPD/issues/2502
-[#2503]: https://github.com/CMSgov/eAPD/issues/2503
-[#2509]: https://github.com/CMSgov/eAPD/issues/2509
-[#2521]: https://github.com/CMSgov/eAPD/issues/2521
```

Comment:
Looks like there's some text in `/Users/rd/fearless/eapd/web/src/i18n/locales/en/apd/stateProfile.yaml` that still refers to "State Cost Categories"

Reply:
oh thanks, I'll check it out and I'll change the route too

Reply:
actually I'm going to leave the route, because the next one is contractor-costs, I think leaving it lines up better